src/ansys/geometry/core/designer/geometry_commands.py

```diff
@@ -1138,3 +1139,24 @@ def split_body(
             design._update_design_inplace()
 
         return result.success
+
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def get_round_info(self, face: "Face") -> tuple[bool, Real]:
+        """Get info on the rounding of a face.
+
+        Parameters
+        ----------
+        Face
+            The design face to get round info on
+
+        Returns
+        -------
+        bool
+            Direction of circular cross-section
+        Real
+            Radius of the round
+        """
```

Comment:
Your return section should be a tuple with a boolean and a Real. Also, let's explain a bit better the boolean... It doesn't make sense that a boolean represents a direction (at least it is not straightforward).